src/Service.Tests/CosmosTests/TestBase.cs

```diff
@@ -56,6 +60,12 @@ public class TestBase
     id : ID,
     name : String,
     details : String
+}
+
+type Earth @model(name:""Earth"") {
+    id : ID,
+    name : String,
+    type: String @authorize(policy: ""authenticated"")
```

Comment:
This is not the correct usage of the authorize directive.

Expected Usage: 
- `@authorize(roles: "authenticated")
- `@authorize(roles: [ "authenticated", "roleName" ])`

[Reference- Hot Chocolate 12 docs](https://chillicream.com/docs/hotchocolate/v12/security/authorization#roles)

Reply:
The usage of `type Moon @model(name:""Moon"") @authorize(policy: ""Crater"") {` was to test a failure scenario for the GraphQLAuthorizationHandler class. When using "Policy" within the authorization directive, an automatic AuthorizeResult.NotAllowed will be returned, failing the request.

Reply:
Updated with roles as mentioned in the other reply.